src/main/src/test/java/org/geoserver/logging/LoggingStartupContextListenerTest.java

```diff
@@ -36,6 +43,22 @@
  */
 public class LoggingStartupContextListenerTest {
 
+    static Locale systemLocale;
+
+    @BeforeClass
+    public static void initialize() {
+        systemLocale = Locale.getDefault();
+        Locale.setDefault(Locale.ENGLISH);
+        // reset static state leftover from other tests
+        System.clearProperty(LoggingUtils.RELINQUISH_LOG4J_CONTROL);
+        LoggingUtils.relinquishLog4jControl = false;
```

Comment:
Nicely done!

So this either means something is setting that concurrently (and it may fail). Or something failed to restore the value after completing it's test....